### PR TITLE
Translation to Arabic

### PR DESCRIPTION
**Pull request by celyes**
Translated all files in about directory to Arabic except for `working-groups.md`
Refs: @InterstellarClub/nodejs.ar
/cc @nodejs/nodejs-ar

### CODE REVIEW

**Comment by celyes** [comment]
@Aissaoui-Ahmed @InterstellarClub

**Comment by Maledong** [comment]
@celyes：It would be better if you merge all of these changes together into one, because there're too many changed submitted :)

File: .htmllintrc [hunk 1]
```diff
@@ -8,6 +8,7 @@
   "img-req-alt": false,
   "indent-style": "spaces",
   "indent-width": 2,
+  "end-style": "crlf",
```
Comment (fhemberger) [inline .htmllintrc]: Please keep the default end-style

File: .gitignore [hunk 2]
```diff
@@ -9,3 +9,6 @@ npm-debug.log
 .DS_Store
 .AppleDouble
 .LSOverride
+
+# VS Code rtl markdown stylesheet
+style.css
```
Comment (fhemberger) [inline .gitignore]: Why is the CSS file ignored?

File: locale/ar/about/working-groups.md [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+
+The Docker Working Group's purpose is to build, maintain, and improve official
+Docker images for the Node.js project.
+
+Responsibilities include:
+* Keeping the official Docker images updated in line with new Node.js releases.
+* Decide and implement image improvements and/or fixes.
+* Maintain and improve the images' documentation.
+
+<<<<<<< HEAD
```
Comment (fhemberger) [inline locale/ar/about/working-groups.md]: Please fix Git merge annotations

File: locale/ar/about/working-groups.md [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+Responsibilities include:
+* Translating any Node.js materials they believe are relevant to their
+  community.
+* Reviewing processes for keeping translations up to date and of high quality.
+* Managing and monitoring social media channels in their language.
+* Promoting Node.js speakers for meetups and conferences in their language.
+
+Each language community maintains its own membership.
+
+<<<<<<< HEAD
```
Comment (fhemberger) [inline locale/ar/about/working-groups.md]: Please fix Git merge annotations

**Review by fhemberger — CHANGES_REQUESTED** [review]
Please delete all files in `locale/ar/blog`, blog posts are not translated.
The knowledge base (`locale/ar/knowledge/`) is currently not shown on the website, so you can go easy on that as well.

File: .gitignore (hunk 2 above)
Comment (celyes) [reply .gitignore]: @fhemberger that's just a CSS file I created for VS Code markdown reader (RTL).

File: .htmllintrc (hunk 1 above)
Comment (celyes) [reply .htmllintrc]: done ^_^

File: locale/ar/about/working-groups.md (hunk 3 above)
Comment (celyes) [reply locale/ar/about/working-groups.md]: @fhemberger  Concerning `locale/ar/about/working-groups.md`, we had a problem in the links that lead to the different sections of the page and we created an issue about it 
( #1973 ), so we decided not  to start translating it or pushing it until the problem is fixed.
@Aissaoui-Ahmed @InterstellarClub

**Comment by ghost** [comment]
@celyes：It would be better to merge all of your changes so that others can see clearly what you've changed in the end.

**Comment by celyes** [comment]
> @celyes：It would be better if you merge all of these changes together into one, because there're too many changed submitted :)

you mean squash commits ? @Maledong

File: package.json [hunk 5]
```diff
@@ -58,7 +58,7 @@
     "request": "^2.87.0",
     "require-dir": "1.0.0",
     "semver": "5.5.0",
-    "st": "1.2.2",
+    "st": "^1.2.2",
```
Comment (fhemberger) [inline package.json]: Please revert this change

File: package-lock.json [hunk 6]
```diff
@@ -5486,7 +5486,7 @@
       "dependencies": {
         "bl": {
           "version": "1.2.2",
-          "resolved": "https://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
+          "resolved": "http://registry.npmjs.org/bl/-/bl-1.2.2.tgz",
```
Comment (fhemberger) [inline package-lock.json]: Please revert this change

**Review by fhemberger — CHANGES_REQUESTED** [review]

**Comment by fhemberger** [comment]
Squashing down would be nice to keep @Aissaoui-Ahmed's commit intact. If we squash the PR (which is the default setting), only you are listed as a contributor.

File: package-lock.json (hunk 6 above)
Comment (celyes) [reply package-lock.json]: changes reverted

Comment (fhemberger) [reply package-lock.json]: … now all the blog files are back. 😲

Comment (celyes) [reply package-lock.json]: @fhemberger  my bad :( 
i did it accidentally during the revert process. anyway, I removed them and I removed all the unnecessary files and those that are not translated yet.
thank you for your patience

**Comment by celyes** [comment]
@fhemberger Sorry for the disturb, but would you review the changes for one last time ?

**Review by fhemberger — APPROVED** [review]
Looks good to me from the technical part. If everybody is happy with the translations, we can merge it.

**Comment by celyes** [comment]
Sure ! go ahead @fhemberger

**Comment by fhemberger** [comment]
Done! Thank you and @Aissaoui-Ahmed!
